docs/framework/wcf/samples/ws-2007-federation-http-binding.md

```diff
@@ -105,17 +105,15 @@ This sample demonstrates the use of <xref:System.ServiceModel.WS2007FederationHt
  STS makes a single endpoint available using the standard <xref:System.ServiceModel.WS2007HttpBinding>. The service responds to requests from clients for tokens. If the client is authenticated using a Windows account, the service issues a token that contains the client's user name as a claim. As part of creating the token, STS signs the token using the private key associated with the CN=STS certificate. In addition, it creates a symmetric key and encrypts it using the public key associated with the CN=localhost certificate. In returning the token to the client, STS also returns the symmetric key. The client presents the issued token to the `ICalculator` service and proves that it knows the symmetric key by signing the message with that key.  
   
  When you run the sample, the request for the security token is shown in the STS console window. The operation's requests and responses are displayed in the client and service console windows. Press ENTER in any of the console windows to shut down the application.  
-  
- `Add(100,15.99) = 115.99`  
-  
- `Subtract(145,76.54) = 68.46`  
-  
- `Multiply(9,81.25) = 731.25`  
-  
- `Divide(22,7) = 3.14285714285714`  
-  
- `Press <ENTER> to terminate client.`  
-  
+
+```
+Add(100,15.99) = 115.99
+Subtract(145,76.54) = 68.46
+Multiply(9,81.25) = 731.25
+Divide(22,7) = 3.14285714285714
+Press <ENTER> to terminate client.
+```
+
```

Comment:
I converted this to a single code block instead of a per-line inline code block in order to match the styling of all the other pages where this output is shown